Server/ConcordiaCurriculumManager/Repositories/ObjectSelectors.cs

```diff
@@ -0,0 +1,44 @@
+﻿using ConcordiaCurriculumManager.Models.Users;
+using System.Data;
+using System.Linq.Expressions;
+
+namespace ConcordiaCurriculumManager.Repositories;
+
+public static class ObjectSelectors
+{
+    public static Expression<Func<Role, Role>> RoleSelector() => role => new Role
+    {
+        UserRole = role.UserRole
```

Comment:
Looks good. It might be useful to populate all the info into roles in case it is needed to avoid confusion since the values get assigned wrong default values. But that is not a major concern right now. Same for the other selectors.